### PR TITLE
[IMP] Inventory: updated text for 16 and fixed rst errors

### PR DESCRIPTION
**Pull request by larm-odoo**
Changed the 'create' button references to 'new' for 16, fixed a few extra ":"'s that appeared and messed up the RST code. Removed references to 'Edit' and 'Save' which are not needed in 16.

### CODE REVIEW

**Comment by larm-odoo** [comment]
@odoo/inventory-doc-review  this is also ready for a look when you have a moment- thank you!

File: content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_two_steps.rst [hunk 1]
```diff
@@ -142,11 +141,11 @@ Process a delivery order in two steps (pick + ship)
...
 --------------------
 
-In the :menuselection:`Sales` application, create a new quote by clicking :guilabel:`Create`. Select
+In the :menuselection:`Sales` application, create a new quote by clicking :guilabel:`New`. Select
 (or create) a :guilabel:`Customer`, add a storable :guilabel:`Product` to the order lines, and then
 click :guilabel:`Confirm`.
 
-After confirming the ::abbr:`SO (Sales Order)`, a :guilabel:`Receipt` smart button will appear in
+After confirming the :abbr:`SO (Sales Order)`, a :guilabel:`Receipt` smart button will appear in
 the top-right corner of the :abbr:`PO (Sales Order)` form — click it to reveal the associated
```
Comment (Felicious) [inline content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_two_steps.rst:149]:
```suggestion
the top-right corner of the :abbr:`PO (Purchase Order)` form — click it to reveal the associated
```

**Comment by larm-odoo** [comment]
Thanks for the review, @Felicious! Per our discussion, I am tagging @meng-odoo for the final review since Zac is out, and this is such a small version change (only changed images and a few words).

File: content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_two_steps.rst [hunk 2]
```diff
@@ -59,9 +59,8 @@ Shipments`. :guilabel:`Save` the changes to finish.
    Activating two-step receipts and deliveries will create new *input* and *output* locations, which
    by default, are labeled :guilabel:`WH/Input` and :guilabel:`WH/Output`, respectively, on the
    :guilabel:`Locations` dashboard. To rename these locations, go to :menuselection:`Configuration
-   --> Locations`, select the :guilabel:`Location` to change, and then click :guilabel:`Edit`. On
-   the location form, update the :guilabel:`Location Name`, make any other changes (if necessary)
-   and :guilabel:`Save` the changes.
+   --> Locations`, and select the :guilabel:`Location` to change. On the location form, update the
+   :guilabel:`Location Name`, make any other changes (if necessary).
```
Comment (meng-odoo) [inline content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_two_steps.rst:63]:
```suggestion
   :guilabel:`Location Name`, and make any other changes (if necessary).
```

File: content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_two_steps.rst [hunk 3]
```diff
@@ -142,13 +140,12 @@ Process a delivery order in two steps (pick + ship)
...
-or create a :guilabel:`Customer`, add a storable :guilabel:`Product` to the order lines, and then
+In the :menuselection:`Sales` application, create a new quote by clicking :guilabel:`New`. Select
+(or create) a :guilabel:`Customer`, add a storable :guilabel:`Product` to the order lines, and then
 click :guilabel:`Confirm`.
 
-After confirming the :abbr:`SO (Sales Order)`, a :guilabel:`Delivery` smart button will appear in
-the top-right corner of the :abbr:`SO (Sales Order)` form. Click it to reveal the associated
-delivery forms.
+After confirming the :abbr:`SO (Sales Order)`, a :guilabel:`Receipt` smart button will appear in
+the top, above the :abbr:`SO (Sales Order)` form — click it to reveal the associated receipt.
```
Comment (meng-odoo) [inline content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_two_steps.rst:148]:
```suggestion
the top, above the :abbr:`SO (Sales Order)` form—click it to reveal the associated receipt.
```

**Review by meng-odoo — CHANGES_REQUESTED** [review]
Hi @larm-odoo, this looks good to me! I had a couple tiny changes that you can easily commit using the new method hehe :) Please go ahead and tag Sam for technical review afterwards!

File: content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_two_steps.rst (hunk 3 above)
Comment (larm-odoo) [reply content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_two_steps.rst:148]: I realized after doing the images for the new Milk design- it is a DELIVERY smart button, not a receipt! I changed it so it is correct.

**Comment by larm-odoo** [comment]
Hi @samueljlieber - per Meng, this is ready for ya!

**Review by samueljlieber — APPROVED** [review]
Hi @larm-odoo the changes looks great to me, nice job! 

@StraubCreative on to you!

**Review by StraubCreative — APPROVED** [review]
LGTM 👍 

@robodoo r+

**Pull request closed without merging**